test/helpers.ts

```diff
@@ -99,47 +89,6 @@ export function runQuery(databaseName: string, queryString: string, assertFn?: (
   }
 }
 
-async function cleanDB(modules: string[], region: string | undefined): Promise<void> {
-  const dbAlias = `cleandb${Date.now()}`;
-  const awsRegion = region ?? process.env.AWS_REGION ?? 'barf';
-  logger.info(`Cleaning ${dbAlias} in ${awsRegion}...`);
-  await iasql.connect(dbAlias, 'not-needed', 'not-needed');
-  logger.info('DB created...');
-  await iasql.install(['aws_account'], dbAlias, config.db.user);
-  const conn = await createConnection({
-    name: dbAlias,
-    type: 'postgres',
-    username: 'postgres',
-    password: 'test',
-    host: 'localhost',
-    port: 5432,
-    database: dbAlias,
-    extra: { ssl: false, },
-  });
-  logger.info(`Connection created...`);
-  await conn.query(`
-    INSERT INTO aws_account (region, access_key_id, secret_access_key)
-    VALUES ('${awsRegion}', '${process.env.AWS_ACCESS_KEY_ID}', '${process.env.AWS_SECRET_ACCESS_KEY}')
-  `);
-  logger.info('AWS Account set up');
-  await iasql.install(modules, dbAlias, config.db.user);
-  logger.info(`Modules ${modules.join(', ')} installed...`);
-  const delQuery = fs.readFileSync(`${__dirname}/sql/delete_records.sql`, 'utf8');
```

Comment:
this sql is used in tutorial integration tests. it is worth checking if it still makes sense to have it be different to the fn in `iasql_platform`?

Reply:
I think so